R-package/README.md

```diff
@@ -30,47 +59,31 @@ PKG_URL <- "https://github.com/microsoft/LightGBM/releases/download/v3.0.0/light
 remotes::install_url(PKG_URL)
 ```
 
-### Installing Precompiled Binaries
-
-Starting with `LightGBM` 3.0.0, precompiled binaries for the R package are created for each release. These packages do not require compilation, so they will be faster and easier to install than packages that are built from source. These packages are created with R 4.0 and are not guaranteed to work with other R versions.
-
-Binaries are available for Windows, Mac, and Linux systems. They are not guaranteed to work with all variants and versions of these operating systems. Please [open an issue](https://github.com/microsoft/LightGBM/issues) if you encounter any problems.
-
-To install a binary for the R package:
-
-1. Choose a release from [the "Releases" page](https://github.com/microsoft/LightGBM/releases).
-2. Choose a file based on your operating system. Right-click it and choose "copy link address".
-    * Linux: `lightgbm-{VERSION}-r40-linux.tgz`
-    * Mac: `lightgbm-{VERSION}-r40-macos.tgz`
-    * Windows: `lightgbm-{VERSION}-r40-windows.zip`
-3. Copy that link into `PKG_URL` in the code below and run it.
+#### Custom Installation (Linux, Mac)
 
-This sample code installs version 3.0.0-1 of the R package on Mac.
+The steps above should work on most systems, but users with highly-customized environments might want to change how R builds packages from source.
 
-```r
-PKG_URL <- "https://github.com/microsoft/LightGBM/releases/download/v3.0.0rc1/lightgbm-3.0.0-1-r40-macos.tgz"
+To change the compiler used when installing the CRAN package, you can create a file `~/.R/Makevars` which overrides `CC` (`C` compiler) and `CXX` (`C++` compiler).
 
-local_file <- paste0("lightgbm.", tools::file_ext(PKG_URL))
+For example, to use `gcc` instead of `clang` on Mac, you could use something like the following:
 
-download.file(
-    url = PKG_URL
-    , destfile = local_file
-)
-install.packages(
-    pkgs = local_file
-    , type = "binary"
-    , repos = NULL
-)
+```make
+# ~/.R/Makevars
+CC=gcc-8
+CXX=g++-8
+CXX11=g++-8
 ```
 
-### Preparation
+### Installing from Source with CMake <a name="install"></a>
 
 You need to install git and [CMake](https://cmake.org/) first.
 
-Note: 32-bit (i386) R/Rtools is currently not supported.
+Note: this method is only supported on 64-bit systems. If you need to run LightGBM on 32-bit Windows (i386), follow the instructions in ["Installing the CRAN Package"](#installing-the-cran-package).
```

Comment:
Is it still true?..

Reply:
yes, we haven't added support for a 32-bit build with CMake and I don't think we should.